src/engine/source/server/src/endpoints/baseEndpoint.hpp

```diff
@@ -11,6 +11,7 @@
 #define _BASE_ENDPOINT_H
 
 #include <string>
+#include "metrics.hpp"
```

Comment:
shouldn't this reference go in the .cpp file? and should we use <> instead of ""?

Reply:
+1